core/src/com/projectkorra/projectkorra/util/LightManager.java

```diff
@@ -295,63 +283,12 @@ public void restart() {
         startLightReverter();
     }
 
-    private static class LightData implements Comparable<LightData> {
-        private final Location location;
-        private final int brightness;
-        private final Collection<? extends Player> observers;
-        private final long expiryTime;
-
-        private LightData(Location location, int brightness, Collection<? extends Player> observers, long expiryTime) {
-            this.location = location;
-            this.brightness = brightness;
-            this.observers = observers;
-            this.expiryTime = expiryTime;
-        }
-
-        /**
-         * Calculates the hash code for this object. The hash code is based on the
-         * values of the location, brightness, observers, and expiryTime fields.
-         *
-         * @return the hash code of this object
-         */
-        @Override
-        public int hashCode() {
-            return Objects.hash(location, brightness, observers, expiryTime);
-        }
-
-        /**
-         * Checks if this LightData object is equal to another object. Two LightData objects are considered
-         * equal if they have the same brightness, expiryTime, location, and observers.
-         *
-         * @param obj the object to compare this LightData object to
-         * @return true if the objects are equal, false otherwise
-         */
-        @Override
-        public boolean equals(Object obj) {
-            if (this == obj) return true;
-            if (obj == null || getClass() != obj.getClass()) return false;
-            LightData that = (LightData) obj;
-            return brightness == that.brightness &&
-                    expiryTime == that.expiryTime &&
-                    location.equals(that.location) &&
-                    observers.equals(that.observers);
-        }
-
-        /**
-         * Returns a string representation of the LightData object.
-         *
-         * @return a string in the format "LightData{location=..., brightness=..., observers=..., expiryTime=...}"
-         */
-        @Override
-        public String toString() {
-            return "LightData{" +
-                    "location=" + location +
-                    ", brightness=" + brightness +
-                    ", observers=" + observers +
-                    ", expiryTime=" + expiryTime +
-                    '}';
-        }
-
```

Comment:
Is it necessary to remove these functions?

Reply:
I mean it's not necessary to *remove* them, but having them does nothing that isn't replicated by default functionality of a record